addons/l10n_it_edi/models/account_move.py

```diff
@@ -744,6 +744,10 @@ def _l10n_it_edi_import_invoice(self, invoice, data, is_new):
 
         self.move_type = move_type
 
+        # the journal might've changed, so we need to recompute the name in case it was set (first entry in journal)
+        self.name = False
+        self._compute_name()
```

Comment:
```suggestion
    if self.name and self.name != '/':
        # the journal might've changed, so we need to recompute the name in case it was set (first entry in journal)
        self.name = False
        self._compute_name()
```

Reply:
Good point, done!
Thanks.